packages/backend-core/API.md

```diff
@@ -170,6 +171,24 @@ const organization = await clerkAPI.organizations.createOrganization({
 });
 ```
 
+#### updateOrganizationMetadata(params)
+
+Update an organization's metadata attributes by merging existing values with the provided parameters.
+
+You can remove metadata keys at any level by setting their value to `null`.
+
+Available parameters are:
+
+- _publicMetadata_ Metadata saved on the organization, that is visible to both your Frontend and Backend APIs.
+- _privateMetadata_ Metadata saved on the organization, that is only visible to your Backend API.
+
+```js
+const organization = await clerkAPI.organizations.updateOrganizationMetadata({
+  publicMetadata: { public: 'metadata' },
```

Comment:
🙃 
This might just be me but this example _(also in the tests)_ seems kind of confusing :P 
E.g. `publicMetadata: { hello: 'world' },` would be more than enough. With this example I felt that I would need to pass the `metadata` inside the `public` parameter. It might be just me though 🤷

Reply:
Thanks for the comment. I can see how this might be confusing. Updated!